sway-lib-std/src/prelude.sw

```diff
@@ -17,3 +17,6 @@ use ::assert::assert;
 use ::option::Option;
 use ::result::Result;
 use ::revert::{require, revert};
+
+/* Convert */
```

Comment:
Is this comment needed? NOne of the other modules have comments.

Reply:
So blocks of imports have comments:
![image](https://user-images.githubusercontent.com/59666792/199529016-792ceccc-bee3-4997-8659-8197cf5b1885.png)
I'm hoping we'll have an `Into` trait in the future that will live under `/* Convert */` as well.